components/emoji_picker/components/emoji_picker_item.tsx

```diff
@@ -71,15 +71,17 @@ function EmojiPickerItem({emoji, rowIndex, isSelected, onClick, onMouseOver}: Pr
     const emojiUnified = emoji.unified ? emoji.unified.toLowerCase() : emoji.name.toLowerCase();
 
     return (
-        <div className={itemClassName}>
+        <div
+            className={itemClassName}
+            onClick={handleClick}
+        >
             <div data-testid='emojiItem'>
                 <img
                     alt={'emoji image'}
                     data-testid={emoji.short_names}
                     onMouseOver={throttledMouseOver}
```

Comment:
what are you thought on moving `onMouseOver` too

Reply:
Yeah, that'd make sense. I'll do that as well